modules/develop/pages/data-transforms/build.adoc

```diff
@@ -0,0 +1,655 @@
+
+[tabs]
+======
+Go::
++
+--
+[source,go]
+----
+package main
+
+import (
+  "fmt"
+  "os"
+	"github.com/redpanda-data/redpanda/src/transform-sdk/go/transform"
+)
+
+func main() {
+	inputTopic, ok := os.LookupEnv("REDPANDA_INPUT_TOPIC")
+  if ok {
+    fmt.Printf("Input topic: %s\n", inputTopic)
+  } else {
+    fmt.Println("An input topic is not set")
+  }
+
+  outputTopic0, ok := os.LookupEnv("REDPANDA_OUTPUT_TOPIC_0")
+  if ok {
+    fmt.Printf("Output topic 0: %s\n", outputTopic0)
+  } else {
+    fmt.Println("An output topic is not set")
+  }
+
+  outputTopic1, ok := os.LookupEnv("REDPANDA_OUTPUT_TOPIC_1")
+  if ok {
+    fmt.Printf("Output topic 1: %s\n", outputTopic1)
+  } else {
+    fmt.Println("Only one output topic is set")
+  }
+
+	transform.OnRecordWritten(doNothing)
+}
+
+// Required callback, even if it does nothing
+func doNothing(event transform.WriteEvent, writer transform.RecordWriter) error {
+	return nil
+}
+----
+--
+Rust::
++
+--
+[source,rust]
+----
+use redpanda_transform_sdk::*;
+use std::env;
+use log::error;
+
+fn main() {
+  // Set up logging
+  env_logger::init();
+
+  // Access built-in environment variables
+  match env::var("REDPANDA_INPUT_TOPIC") {
+    Ok(input_topic) => println!("Input topic: {}", input_topic),
+    Err(_) => println!("An input topic is not set"),
+  }
+
+  match env::var("REDPANDA_OUTPUT_TOPIC_0") {
+    Ok(output_topic_0) => println!("Output topic 0: {}", output_topic_0),
+    Err(_) => println!("An output topic is not set"),
+  }
+
+  match env::var("REDPANDA_OUTPUT_TOPIC_1") {
+    Ok(output_topic_1) => println!("Output topic 1: {}", output_topic_1),
+    Err(_) => println!("Only one output topic is set"),
+  }
+
+  on_record_written(do_nothing);
+}
+
+// Required callback, even if it does nothing
+fn do_nothing(_event: WriteEvent, _writer: &mut RecordWriter) -> anyhow::Result<()> {
+  Ok(())
+}
+----
+--
+======
+
+=== Connect to the Schema Registry
+
+You can use the Schema Registry client library to read and write schemas as well as serialize and deserialize records. This client library is useful when working with schema-based topics in your data transforms.
+
+See also:
+
+- xref:manage:schema-reg/schema-reg-overview.adoc[].
+- xref:reference:data-transforms/golang-sdk.adoc[Go Schema Registry client reference]
+- xref:reference:data-transforms/rust-sdk.adoc[Rust Schema Registry client reference]
+
+==== Initialize the Schema Registry client
+
+First, initialize the Schema Registry client. This client will interact with the local Redpanda Schema Registry.
+
+[tabs]
+======
+Go::
++
+--
+[source,go]
+----
+package main
+
+import (
+  "context"
+  "fmt"
+	"github.com/redpanda-data/redpanda/src/transform-sdk/go/transform"
+	"github.com/redpanda-data/redpanda/src/transform-sdk/go/transform/sr"
+)
+
+func main() {
+  client := sr.NewClient()
+	transform.OnRecordWritten(doNothing)
+
+  // Use the client...
```

Comment:
You can't use the client here, `transform.OnRecordWritten` runs forever and never returns.

This should be passed into the callback, or registered as a global variable (same for Rust).